_posts/2021-01-25-solidity-developer-survey-2020-results.md

```diff
@@ -0,0 +1,359 @@
+- **Solidity Experience**: 43% of respondents rate themselves Solidity experts with an expertise level of 8 (out of 10) or higher.
+- **Solidity Developer Experience**: While noting many things that could still improve, there was an overall positive/good sentiment towards Solidity. Most respondents believed that the Solidity developer experience had some or much improvement over the last year.
+- **Solidity's Challenges**: The biggest challenge or problem in developer experience for most respondents is the availability and quality of tooling. Also seen as critical aspects in order to foster adoption are error handling, IDE integrations and better documentation.
+- **Most Liked, Dreaded and Anticipated**: Topics with room for imporovement that came up a lot: Error handling, debugging, the need for more tools and libraries, breaking dependencies.
+- **Contributing and Language Design**: There was surprisingly little interest and engagement in language design topics, partly because developers didn't know how to get involved, partly because they were not interested. (We will work on this! Efforts to foster collaborative language design are under way, we'll share more soon.)
+- **Community**: Almost 17% of participants in this survey do not interact with other Solidity developers at all!
+
+### (1) Survey Audience and Developer Profiles
+
+Firstly, let's have a look at the developers that participated in the survey. In total, we received **193 responses from 48 different countries**. 
+
+![Survey Participants World Map](/img/2021/01/world.png)
+
+#### Residency
+
+⚠️ _Note that the fact that this survey has only been shared in English is an important factor to consider when interpreting results regarding language and country of residency._
+
+The respondents stated to live in Algeria, Argentina, Australia, Austria, Belgium, Brazil, Bulgaria, Canada, China, Costa Rica, Denmark, Finland, France, Germany, Ghana, Greece, Honduras, Hong Kong, India, Indonesia, Iran, Ireland, Israel, Italy, Japan, Lithuania, Malaysia, Mexico, Netherlands, New Zealand, Nigeria, Pakistan, Philippines, Poland, Portugal, Romania, Russia, Serbia, Singapore, South Africa, Spain, Switzerland, Taiwan, Thailand, Timor-Leste, Ukraine, United Kingdom and United States of America. 
+
+With roughly 20%, the big majority of respondents live in the United States of America, followed by India (8%) and Germany (7%).
+
+![Survey Participants List of Countries](/img/2021/01/residency.png)
+
+#### Language
+
+The developers speak a wide variety of different languages as their primary language, with English being by far the most popular one. German, Spanish, Russian and French follow with similar levels.
+
+![Primary Language](/img/2021/01/langprimary.png)
```

Comment:
Why is this one sorted by language instead of "bar length"?

Reply:
changed the language graphs to be sorted by length